src/features/remove/index.js

```diff
@@ -1,15 +1,20 @@
+import SelectionModule from '../selection';
+
+import { KeypressBehavior } from './behaviors/KeypressBehavior';
 import { RemoveElementsBehavior } from './behaviors/RemoveElementsBehavior';
 import { RemoveElementsCommand } from './commands/RemoveElementsCommand';
 import { RemoveProvider } from './providers/RemoveProvider';
 
 
 export default {
     __depends__: [
+        SelectionModule,
     ],
     __init__: [
         'remove',
     ],
     __behaviors__: [
+        [ 'keypress', KeypressBehavior ],
```

Comment:
For defining multiple keyboard events for the same behavior we could just list them up like this:

```
[ [ 'del' ], KeypressBehavior ],
[ [ 'backspace' ], KeypressBehavior ],
```